### PR TITLE
Product Update Love Cycle April 2024

### PR DESCRIPTION
**Pull request by jayk-u**
Here's the latest Love Cycle Product Update.
The main goal was to get something close to traditional patch notes (as seen... everywhere like video games).
The tone is lighter, to match our approach of these cycles and to add our human touch.
Links to specific part were added when possible, suggested a screenshot for highlights but opened to propositions.

I'd like to get developers reviews as I'm speaking directly of their work here. Thanks!

### CODE REVIEW

File: src/_product-updates/2024-04-05-love-cycle.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+![new-dashboard-2024.jpg](/images/changelog/love-cycle.png)
+
+We have a tradition of scheduling what we call Love Cycles between our development cycles: times dedicated to improving Bump.sh, responding to specific user requests that we find relevant, and providing a general polish.
+
+Our last Love Cycle has recently ended, and we wanted to share with you all the small changes we've made. If you have a feature in mind or a need, don't hesitate to share your feedback with us at hello (at) bump.sh!
+
+Improvements:
+
+- Overlay Support: [Overlays](https://docs.bump.sh/help/specification-support/overlays/) are a new feature (beta) of OpenAPI that allow applying a layer over an API document without modifying it, to adapt to a specific context. Now supported by Bump.sh, feel free to read our [recent guide](https://docs.bump.sh/guides/openapi/augmenting-generated-openapi/) on the topic!
+- We added a [new command to our CLI](https://github.com/bump-sh/cli?tab=readme-ov-file#bump-overlay-definition_file-overlay_file) that allows applying an overlay to documentation.
```
Comment (ChristopheDujarric) [inline src/_product-updates/2024-04-05-love-cycle.md:16]: This is two-in-one in reality. The CLI lets users merge definition files and overlay files, into a new definition file that they can deploy as they used to.

File: src/_product-updates/2024-04-05-love-cycle.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- Examples will now display correctly regardless of the chosen format. Bonus for those in XML which will also display the appropriate highlight.
+- [OpenAPI] Improved support for externalValue which now displays a link to the designated resource.
```
Comment (ChristopheDujarric) [inline src/_product-updates/2024-04-05-love-cycle.md:18]: I guess there the link to the OpenAPI specification page missing?

File: src/_product-updates/2024-04-05-love-cycle.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- Navigation has been improved for logged-in users, by adding buttons for quick return to the Dashboard.
+- From now on, the timestamp of your deploys and other actions will be based on your browser's timezone instead of GMT+1 by default.
+- We have added and modified some warning messages to make them more explicit.
+
+![new-highlights.png](/images/changelog/new-highlights.png)
+
+Fixes:
+- We resolved the issue with discriminator values not always mapping correctly in certain specific cases.
+- The generation of some API keys was not always going smoothly, but this is now fixed.
+- [AsyncAPI] The binding support is now properly applied at the channel level.
```
Comment (ChristopheDujarric) [inline src/_product-updates/2024-04-05-love-cycle.md:32]: Missing link here, too?

**Review by ChristopheDujarric — CHANGES_REQUESTED** [review]

File: src/_product-updates/2024-04-05-love-cycle.md (hunk 3 above)
Comment (jayk-u) [reply src/_product-updates/2024-04-05-love-cycle.md:32]: Same here than last comment.

File: src/_product-updates/2024-04-05-love-cycle.md (hunk 1 above)
Comment (paulRbr) [reply src/_product-updates/2024-04-05-love-cycle.md:16]: :+1: agreed it's the same news as the previous point. Maybe a sentence talking about the cli, something like “feel free to read our recent guide on the topic and poke around with the new `bump overlay` CLI command!”

File: src/_product-updates/2024-04-05-love-cycle.md (hunk 2 above)
Comment (paulRbr) [reply src/_product-updates/2024-04-05-love-cycle.md:18]: If you only want a highlight, there's a ==markdown syntax for that== (see [our own help page about markdown](https://docs.bump.sh/help/enhance-documentation-content/markdown-support/#common-markdown-syntax-support))

File: src/_product-updates/2024-04-05-love-cycle.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Quality of Life:
+- We have improved the appearance of highlights during navigation on documentation, to make them more visible and comfortable.
+- Navigation has been improved for logged-in users, by adding buttons for quick return to the Dashboard.
+- From now on, the timestamp of your deploys and other actions will be based on your browser's timezone instead of GMT+1 by default.
+- We have added and modified some warning messages to make them more explicit.
+
+![new-highlights.png](/images/changelog/new-highlights.png)
+
+Fixes:
+- We resolved the issue with discriminator values not always mapping correctly in certain specific cases.
```
Comment (paulRbr) [inline src/_product-updates/2024-04-05-love-cycle.md:30]: I would add a “Special thanks to Phil Sturgeon for pointing it out and having a thorough discussion about discriminators with the team”

**Review by paulRbr — COMMENTED** [review]
I left a few comments, but overall it looks good. Thanks Jay!

File: src/_product-updates/2024-04-05-love-cycle.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+![new-dashboard-2024.jpg](/images/changelog/love-cycle.png)
+
+We have a tradition of scheduling what we call Love Cycles between our development cycles: times dedicated to improving Bump.sh, responding to specific user requests that we find relevant, and providing a general polish.
+
+Our last Love Cycle has recently ended, and we wanted to share with you all the small changes we've made. If you have a feature in mind or a need, don't hesitate to share your feedback with us at hello (at) bump.sh!
+
+Improvements:
+
+- Overlay Support: [Overlays](https://docs.bump.sh/help/specification-support/overlays/) are a new feature (beta) of OpenAPI that allow applying a layer over an API document without modifying it, to adapt to a specific context. Now supported by Bump.sh, feel free to read our [recent guide](https://docs.bump.sh/guides/openapi/augmenting-generated-openapi/) on the topic! We also updated our CLI accordingly with a [new command](https://github.com/bump-sh/cli?tab=readme-ov-file#bump-overlay-definition_file-overlay_file).
```
Comment (ChristopheDujarric) [inline src/_product-updates/2024-04-05-love-cycle.md:15]:
```suggestion
- Overlay Support: [Overlays](https://docs.bump.sh/help/specification-support/overlays/) are a new feature (beta) of OpenAPI that allows applying a layer over an API document without modifying it, to adapt to a specific context. Now supported by the Bump.sh CLI with a [new command](https://github.com/bump-sh/cli?tab=readme-ov-file#bump-overlay-definition_file-overlay_file). Feel free to read our [recent guide](https://docs.bump.sh/guides/openapi/augmenting-generated-openapi/) on the topic!
```

**Comment by ChristopheDujarric** [comment]
One last suggestion for you Jay!

**Review by ChristopheDujarric — APPROVED** [review]

**Pull request merged**